source/_changelogs/2017-08-01-August.md

```diff
@@ -0,0 +1,27 @@
+---
+title: August 2017
+changelog: true
+---
+
+## Platform Improvements
+### Global CDN Surrogate Key Cache Clearing API Improvement 
```

Comment:
How about including links to https://wordpress.org/plugins/pantheon-advanced-page-cache/ and https://drupal.org/project/pantheon_advanced_page_cache ?